Car/src/activity/my-profile/my-profile-activity/cars/car-activity/edit-cars/EditCars.tsx

```diff
@@ -75,17 +53,58 @@ const EditCars = (navigation: any) => {
     );
 
     const [plateNumber, setPlateNumber] = useState<string>("");
-
+    const [isValidPlateNumber, setValidPlateNumber] = useState(true);
     const [photo, setPhoto] = useState({} as ImagePickerResponse);
 
-    const [loading, setLoading] = useState(false);
+    let modelPickerController: any;
+    let brandPickerController: any;
+    let colorPickerController: any;
 
     useEffect(() => {
-        BrandService.getBrands().then((res) => {
-            setBrands(res.data);
+        BrandService.getBrands().then((response) => {
+            setBrands(response.data);
         });
+        let carId = Number(navigation.props.route.params.carId);
+
+        CarService.getById(carId).then((response) => {
+            const car = response.data;
+            const carModel = car?.model;
+            let carBrandItem : CarDropDownPickerItem = {
+                label: carModel?.brand?.name ?? "",
+                value: carModel?.brand?.id.toString() ?? ""
+            };
+            let carModelItem : CarDropDownPickerItem = {
+                label: carModel?.name ?? "",
+                value: carModel?.id.toString() ?? ""
+            };
+            const carColor = colors.find(obj => {
+                return obj.value === car?.color.toString();
+            });
+
+            selectBrandHandle(carBrandItem);
+            setColor(carColor!);
+            setPlateNumber(car?.plateNumber ?? "");
+            setModel(carModelItem);
+        }).catch((e: any) => console.log(e));
     }, []);
 
+    function validatePlateNumber () {
+        let isValid;
+
+        if (
+            !plateNumber ||
+            plateNumber.length < MIN_PLATE_NUMBER_LENGTH ||
+            plateNumber.length > MAX_PLATE_NUMBER_LENGTH ||
+            !plateNumber.match(/^[A-ZА-Я0-9-]+$/
+            )
+        ) {
+            isValid = false;
+        } else {
+            isValid = true;
+        }
+        setValidPlateNumber(isValid);
+    }
+
```

Comment:
unnecessary branching and variable declarations

Reply:
Done!